Source/AliveLibAO/Abe.cpp

```diff
@@ -5361,7 +5365,27 @@ void Abe::State_136_ElumMountEnd_42E110()
 
 void Abe::State_137_ElumUnmountBegin_42E2B0()
 {
-    NOT_IMPLEMENTED();
+    if (!field_104_pending_resource_count && field_10_anim.field_4_flags.Get(AnimFlags::eBit18_IsLastFrame))
+    {
+        while (!ResourceManager::FreeResource_455550(field_1A4_resources.res[58]))
+        {
+            // Empty
+        }
+
+        field_1A4_resources.res[58] = {};
+        field_104_pending_resource_count = 3;
+
+        ResourceManager::LoadResourceFile_4551E0("ABEBSIC.BAN", reinterpret_cast<ResourceManager::TLoaderFn>(OnResourceLoaded_4019A0), reinterpret_cast<Camera*>(this), 0);
```

Comment:
Use the template overload:

 ResourceManager::LoadResourceFile("ABENOELM.BND", BaseAliveGameObject::OnResourceLoaded_4019A0, this);

Then you can drop the casts, you can remove the last 0 argument as it defaults to in this template helper, same for the other LoadResourceFile calls in here